src/Service.Host/client/src/components/parts/tabs/GeneralTab.js

```diff
@@ -177,6 +178,7 @@ function GeneralTab({
                     loading={productAnalysisCodesSearchLoading}
                     fetchItems={searchProductAnalysisCodes}
                     links={false}
+                    disabled={!!salesPartNumber}
```

Comment:
is !! the same as  [nothing]

Reply:
It's a thing I sometimes do to quickly convert any javascript type to a bool. Anything falsy becomes false and anything truthy becomes true. Do you think I should do it more explicitly?

Reply:
No that's pretty cool. I'm so used to just using the variable directly as the truthy thing in js that it looked strange.
Does the component proptypes complain about being passed a string? I suppose it (quite reasonably) will.

Reply:
Exactly... "disabled" expects a bool. The alternative to this is Boolean(value), but !! wins for me since it's less typing